src/Microsoft.SqlTools.ServiceLayer/BatchParser/BatchParserWrapper.cs

```diff
@@ -111,6 +117,67 @@ public sealed class BatchParserWrapper : IDisposable
             return batchDefinitionList;
         }
 
+        private static int GetMaxStartLine(IList<Tuple<int, int>> positions) {
```

Comment:
'{' should be on a separate line by itself in C#... please fix all instances in this PR